go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cert-manager/trust-manager
 
-go 1.20
+go 1.21.1
```

Comment:
Any particular reason for selecting 1.21.1? Why not 1.21.0 (first 1.21.x) or 1.21.3 (current latest).

Reply:
Ah, sorry; this was quite an old change and 1.21.1 was the latest version at that point in time. I updated the PR to 1.21.3.